test/Form_test.js

```diff
@@ -922,6 +928,30 @@ describe("Form", () => {
           })
         );
       });
+
+      it("should reset errors and errorSchema state to initial state after correction and resubmission", () => {
+        const { comp, node } = createFormComponent({
+          schema,
+          onError: () => {},
```

Comment:
Why onError empty ?

Maybe this function should be called once, and thus add an expectation for that.